src/views/sponsorship.html

```diff
@@ -12,7 +12,7 @@
             <div id="intro-text" class="unit column grows">
                 <h1 id="hack-cambridge-wordmark">Hack Cambridge<br><span class="pixel">Ternary</span></h1>
                 <p class="tight">
-                    In January 2018, 300 top computer scientists will come together for 24 hours to push the boundaries of technology.
+                    In January 2019, 300 top computer scientists will come together for 24 hours to push the boundaries of technology.
```

Comment:
This feels misleading underneath the "Hack Cambridge Ternary" logo — it seems that HC 2019 is "Hack Cambridge Ternary".  We should make this generic, or update the page in a similar way to the splash page with past tense and a note about next year.

Reply:
Good point. Another possibility would be to have a generic HC logo for the periods between the event and the rebrand. This would also prevent sponsorship from being blocked by design over the summer too.

Reply:
@henry-thompson you should probably see this too, will add you as a reviewer

Reply:
@jmalcic I think that would be a good long-term solution.  For this pull, could we update just the tense / add a note or leave the sponsorship page unchanged so that we can deploy the changes to the splash page?